cmd/tk/bootstrap_github.go

```diff
@@ -118,6 +124,17 @@ func bootstrapGitHubCmdRun(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	// add teams to org repository
+	if !ghPersonal {
+		for _, team := range ghTeams {
+			if err := addGitHubTeam(ctx, ghHostname, ghOwner, ghRepository, ghToken, team, ghDefaultPermission); err != nil {
+				logFailure(err.Error())
```

Comment:
I would expect this to return as soon as it runs into an error, as the current code could result in no-one getting granted access to the repository while the bootstrap continues.

Reply:
Yes that was the intention as you can't make the operations transactional I opted for best-effort. Note that the user that runs the command has full access to the repo.

Reply:
Ack. In that case I would expect a partly successful bootstrap run to return a non-zero exit code as an indicator.

Reply:
Done!